controllers/user/controllers/account_controller.go

```diff
@@ -44,41 +44,42 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const ACCOUNTNAMESPACEENV = "ACCOUNT_NAMESPACE"
+const DEFAULTACCOUNTNAMESPACE = "sealos-system"
```

Comment:
can we switch account to cluster scope?

Reply:
sure，user only have one account，i agree you

Reply:
but not in a hurry，what good if account change to cluster scope？